test.js

```diff
@@ -1,6 +1,5 @@
 const tape = require('tape')
-const rimraf = require('rimraf')
```

Comment:
Hmm, rimraf wasn't in the dependencies?

Reply:
Nope, I think it worked because it was part of `level-test`.